readthedocs/proxito/views/serve.py

```diff
@@ -87,6 +93,14 @@ def get(self,
         ]):
             return self.system_redirect(request, final_project, lang_slug, version_slug, filename)
 
+        # Handle `/projects/subproject` URL redirection
+        if all([
+                final_project.single_version,
```

Comment:
I'm not sure to understand why we are only checking this case when its parent project is Single Version. Why not when it's not?

Reply:
We only check this case when it's at the end of a URL, which isn't true for any other URL pattern.

Reply:
Ah! OK. It wasn't working previously because we were forcing a trailing `/` in the URL to match the regex!